### PR TITLE
setup installer manifest content

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton ptal

File: manifest/imagestream.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+kind: ImageStream
```
Comment (smarterclayton) [inline manifest/imagestream.yaml]: rename to `image-references` (entire filename)

**Comment by bparees** [comment]
@legionus fyi

**Comment by smarterclayton** [comment]
Let's try this out

/lgtm

**Pull request merged**